site/content/events/2016-london/program/index.html

```diff
@@ -9,14 +9,30 @@
 platinum: true
 ---
 <% @eventhome = @page.directory.split(File::SEPARATOR)[0..1].join(File::SEPARATOR) %>
+<b><h2>Confirmed Speakers!</h2></b>
+
+<ul>
+	<li><a href="https://twitter.com/RealGeneKim">Gene Kim</a></li>
+	<li><a href="https://twitter.com/jemolesky">Joanne Molesky</a></li>
+	<li><a href="https://twitter.com/cote">Coté</a></li>
+	<li><a href="https://twitter.com/nzthiago">Thiago Almeida</a></li>
+	<li><a href="https://twitter.com/garethr">Gareth Rushgrove</a></li>
+	<li><a href="https://twitter.com/simonvc">Simon Vans-Colina</a></li>
+	<li><a href="https://twitter.com/ClaireAgutter">Claire Agutter</a></li>
+	<li><a href="https://twitter.com/KrisSaxton">Kris Saxton</a></li>
+        <li><a href="https://twitter.com/caseywest">Carey West</a></li>
```

Comment:
You have a merge conflict because of this:

<<<<<<< HEAD
        <li><a href="https://twitter.com/caseywest">Carey West</a></li>
        <li><a href="https://twitter.com/bridgetkromhout">Bridget Kromhout</a></li>
        <li><a href="https://twitter.com/KrisBuytaert">Kris Buytaert</a></li>
=======
        <li><a href="https://twitter.com/caseywest">Casey West</a></li>
>>>>>>> dod/master

Please change `Carey` to `Casey` and then your PR should be mergeable. Thanks.